gremlin-dotnet/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#    or more contributor license agreements.  See the NOTICE file
+#    distributed with this work for additional information
+#    regarding copyright ownership.  The ASF licenses this file
+#    to you under the Apache License, Version 2.0 (the
+#    "License"); you may not use this file except in compliance
+#    with the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing,
+#    software distributed under the License is distributed on an
+#    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#    KIND, either express or implied.  See the License for the
+#    specific language governing permissions and limitations
+#    under the License.
+
+version: "3.7"
+
+services:
+
+  gremlin-server-test-dotnet:
+    container_name: gremlin-server-test-dotnet
+    image: tinkerpop:gremlin-server-test-${GREMLIN_SERVER}
+    build:
+      context: ../
+      dockerfile: docker/gremlin-test-server/Dockerfile
+      args:
+        - GREMLIN_SERVER=${GREMLIN_SERVER}
+    ports:
+      - "45940:45940"
+      - "45941:45941"
+      - "45942:45942"
+      - "4588:4588"
+    volumes:
+      - ${HOME}/.groovy:/root/.groovy
+      - ${HOME}/.m2:/root/.m2
+      - ${ABS_PROJECT_HOME}/gremlin-test/target:/opt/gremlin-test
+
+  gremlin-dotnet-integration-tests:
+    container_name: gremlin-dotnet-integration-tests
+    image: mcr.microsoft.com/dotnet/sdk:6.0
+    volumes:
+      - .:/gremlin-dotnet
+      - ../gremlin-test/features:/gremlin-test/features
+      - ../docker/gremlin-test-server:/gremlin-dotnet/gremlin-test-server
+    environment:
+      - DOCKER_ENVIRONMENT=true
+      - TEST_TRANSACTIONS=true
+    working_dir: /gremlin-dotnet
+    command: >
+      bash -c "apt-get update && apt-get install dos2unix && dos2unix ./gremlin-test-server/wait-for-server.sh
+      && ./gremlin-test-server/wait-for-server.sh gremlin-server-test-dotnet 45940 300 
+      && dotnet test ./Gremlin.Net.sln"
+    depends_on:
```

Comment:
I had tried the long syntax a while back and it didn't work somehow, but I might have overlooked some details. We will take a look at it again, as it will indeed be much simpler without having that `wait-for-server.sh`.

Reply:
condition: service_healthy is now added in [a25300b](https://github.com/apache/tinkerpop/pull/1788/commits/a25300b27cfbf949826ea810dc10059aa7e43f8d). I'm thinking of keeping wait-for-server.sh as an option, so I've not deleted it yet.